browser/ui/brave_tooltips/BUILD.gn

```diff
@@ -19,12 +19,13 @@ source_set("brave_tooltips") {
   ]
 
   if (is_win || is_linux || is_mac) {
+    public_deps = [ "//skia" ]
```

Comment:
why a public dep?

Reply:
I had been trying something out but not needed, removed.